code/game/objects/structures/crates_lockers/crates/wooden.dm

```diff
@@ -15,4 +15,133 @@
+		if("worldwar")
+			new /obj/item/clothing/suit/armor/vest/old(src)
+			new /obj/item/gun/ballistic/rifle/boltaction(src)
+			new /obj/item/ammo_box/a762(src)
+			new /obj/item/grenade/syndieminibomb/concussion(src)
+			new /obj/item/clothing/head/helmet/old(src)
+
+		if("tommy")
+			new /obj/item/gun/ballistic/automatic/tommygun/no_mag(src)
+			new /obj/item/ammo_box/magazine/tommygunm45/empty(src)
+			for(var/i = 0, i < 12, ++i)
+				new /obj/item/ammo_casing/c45(src)
+			new /obj/item/clothing/head/fedora(src)
+			new /obj/item/clothing/under/rank/det(src)
+
+		if("gang")
+			new /obj/item/gun/ballistic/automatic/mini_uzi(src)
+			new /obj/item/toy/crayon/spraycan(src)
+			new /obj/item/clothing/mask/bandana/black(src)
+
+		if("rushn")
+			new /obj/item/gun/ballistic/automatic/pistol/APS(src)
+			new /obj/item/switchblade(src)
+			new /obj/item/clothing/head/ushanka(src)
+			new /obj/item/clothing/under/rank/security/navyblue/russian(src)
+			new /obj/item/reagent_containers/food/drinks/bottle/vodka(src)
+
+		if("detective")
+			new /obj/item/gun/ballistic/revolver/detective(src)
+			new /obj/item/melee/classic_baton(src)
+			new /obj/item/restraints/handcuffs(src)
+			new /obj/item/clothing/under/lawyer/blacksuit(src)
+			new /obj/item/evidencebag/random(src)
+
+		if("wiz_charge")
+			new /obj/item/gun/energy/laser/retro/old(src)
+			new /obj/item/book/granter/spell/charge(src)
+
+		if("wiz_summon_knock")
+			new /obj/item/book/granter/spell/summonitem(src)
+			new /obj/item/book/granter/spell/knock(src)
+
+		if("wiz_blind")
+			new /obj/item/book/granter/spell/blind(src)
+			new /obj/item/clothing/glasses/blindfold(src)
+
+		if("wiz_forcewall_sacredflame")
+			new /obj/item/book/granter/spell/forcewall(src)
+			new /obj/item/book/granter/spell/sacredflame(src)
+			new /obj/item/extinguisher(src)
+
+		if("wiz_smoke_door")
+			new /obj/item/book/granter/spell/smoke(src)
+			new /obj/item/gun/magic/wand/door(src)
+
+		if("wiz_fireball")
+			new /obj/item/gun/magic/wand/fireball(src)
+
+		if ("wiz_suit") 
+			new /obj/item/clothing/suit/space/hardsuit/wizard(src)
+			new /obj/item/tank/jetpack/oxygen/harness(src)
+			new /obj/item/clothing/mask/gas/syndicate(src)
+			new /obj/item/gun/magic/wand(src) // Wand of nothing
+
+		if("cutlass")
+			new /obj/item/melee/transforming/energy/sword/pirate(src)
+			new /obj/item/clothing/glasses/hud/security/sunglasses/eyepatch(src)
+			new /obj/item/twohanded/binoculars(src)
+			new /obj/item/coin/gold(src)
+			new /obj/item/coin/gold(src)
+			new /obj/item/coin/gold(src)
+
+		if("katana")
+			new /obj/item/katana(src)
+			new /obj/item/throwing_star(src)
+			new /obj/item/throwing_star(src)
+			new /obj/item/clothing/mask/gas/space_ninja(src)
+			new /obj/item/clothing/suit/armor/vest/leather(src)
+
+		if("pharaoh")
+			new /obj/item/clothing/head/pharaoh(src)
+			new /obj/item/nullrod/egyptian(src)
+			new /obj/item/soulstone/anybody/chaplain(src)
```

Comment:
give back the construct shell I want my army reeeeeee

Reply:
It doesn't work unless your cult.